### ISSUE TITLE
Support "threshold" for newrelic dashboard widgets

### ISSUE THREAD

**Issue by csMACnz** [terraform-providers/terraform-provider-newrelic#194]
It would be great to be able to specify the threshold values for `newrelic_dashboard` widgets so that these can be driven from our terraform deployments as well.

[In the api](https://docs.newrelic.com/docs/insights/insights-api/manage-dashboards/insights-dashboard-api#schema), this appears to be sent under presentation  as `red`/`yellow`
```
"presentation": {
  "title": "Threshold Event Chart",
  "notes": null,
  "threshold": {
    "red": 18000000,  
    "yellow": 8000000
  }
}
```

On the UI, this presents as `critical`/`warning`

![image](https://user-images.githubusercontent.com/1768046/68433686-04bc2380-021c-11ea-9fe1-566a8517abfa.png)

**Comment by ctrombley** [comment]
Duplicate of #32

**Comment by ctrombley** [comment]
Will PR this once paultyng/go-newrelic#71 is included in a release.

**Comment by ctrombley** [comment]
Addressed in #206
